frontend/test/metabase/scenarios/visualizations/object_detail.cy.spec.js

```diff
@@ -94,6 +94,18 @@ describe("scenarios > question > object details", () => {
       .parent()
       .contains("8");
   });
+
+  it.skip("should not offer drill-through on the object detail records (metabase#20560)", () => {
+    openPeopleTable({ limit: 2 });
+
+    cy.get(".Table-ID")
+      .contains("2")
+      .click();
+    cy.url().should("contain", "objectId=2");
+
+    cy.findByText("Domenica Williamson").click();
+    popover().should("not.exist");
```

Comment:
`popover` doesn't only return a popover element, but also asserts that it's visible. AFAIK when the issue is fixed, `popover` will still break as it won't exist in the DOM at all

Reply:
Oh, good catch.
I didn't know we added that check, and this here is the good example of why we shouldn't add/combine such logic in getters.

Reply:
Fixed https://github.com/metabase/metabase/pull/20579/commits/cd435cf0f769777b4aed721bf4cb1e7cb690f639

Thank you again @kulyk ! <3

Reply:
Actually, that gives us a false positive because Cypress asserts that the element doesn't exist before it even appears. Not a good assertion. I have another one in mind...

Reply:
@kulyk sorry to bother, but can you please take another look at this.
I believe [this is now a fail-safe assertion](https://github.com/metabase/metabase/pull/20579/commits/bf7590b4cb9385c6883869fee040353087e8e68c):
![image](https://user-images.githubusercontent.com/31325167/154494873-f6f713ce-5c1c-4a07-ab9e-6bb4fb0e691a.png)

Reply:
Yeah, good call, looks good to me!